server/events/project_command_builder.go

```diff
@@ -629,8 +629,10 @@ func (p *DefaultProjectCommandBuilder) buildProjectPlanCommand(ctx *command.Cont
 			foundDir := false
 
 			for _, f := range modifiedFiles {
-				if filepath.Dir(f) == cmd.RepoRelDir {
+				// Check if the file is in the directory or any subdirectory of the target
+				if strings.HasPrefix(filepath.Dir(f), cmd.RepoRelDir) {
```

Comment:
Personally I would hide this behind a flag to make sure people that want to be very strict can still do so.

Reply:
@ibalat, yeah, that looks like a good flag to me. Just be aware that I can review this, but I'm not sure I'm the best person to give a final word if this should be accepted.

Reply:
I totally agree with @Fabianoshz. Introducing that flag could help to backwards compatibility 👍

Reply:
okey, thanks for your comments. I'll update the pr